src/airport.js

```diff
@@ -26,10 +26,22 @@ class Airport {
         this.#landedPlanes = array;
     }
 
+    checkPlaneCanLand() {
+        return !this.isAirportFull() && !this.isWeatherStormy();
+    }
+    /* Not overly happy with how I implemented landPlane to include all the conditions before adding a plane to the array
+    Changed it from a single if statement:
```

Comment:
Need to think about the order of execution for the next challenge as I could not really refactor more cleanly.